django/contrib/admin/static/admin/css/widgets.css

```
@@ -460,6 +474,54 @@ span.clearable-file-input label {
     color: white;
 }
 
+.calendar .monthlist caption {
+	padding: 7px;
+}
+
+.calendar .monthlist caption a {
+    font-size: 13px;
```

```suggestion
    font-size: 0.8125rem;
```

We should never set font sizes with absolute units.

---

maybe change all font sized to rem in another pull request, but i replaced my changes to rem unit.